dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+---
+layout: page
+title: Ad Unit Reference
+description: Ad Unit Reference
+top_nav_section: dev_docs
+nav_section: reference
+pid: 10
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Ad Unit Reference
+{:.no_toc}
+
+Correctly configuring the `adUnit` object is of central importance to setting up Prebid.js on your site.
+
+This page is an API reference describing all the features of the `adUnit`.
+
+{: .alert.alert-success :}
+This page is a work in progress.  Over time, we will be pointing all `adUnit`-related documentation at this page.  We are releasing it now in the hopes that it will be useful even in its incomplete state.
+
+* TOC
+{:toc}
+
+## `adUnit`
+
+See the table below for the list of properties on the ad unit.  For example ad units, see the [Examples](#addAdUnits-Examples) below.
+
+
+{: .table .table-bordered .table-striped }
+| Name         | Scope    | Type                                  | Description                                                                                                                                                         |
+|--------------+----------+---------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `code`       | Required | String                                | Unique identifier you create and assign to this ad unit.  Used to set query string targeting on the ad. If using GPT, we recommend setting this to slot element ID. |
+| `sizes`      | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.  For 1.0 and later, prefer one of the `mediaTypes.*.sizes` listed below.    |
+| `bids`       | Required | Array[Object]                         | Each bid represents a request to a bidder.  For a list of properties, see [Bids](#bids) below.                                                                      |
+| `mediaTypes` | Optional | Object                                | Defines the media type of the ad.  For a list of properties, see [Media Types](#mediatypes) below.                                                                  |
```

Comment:
I assume we'll modify this wording later to refer to multiple media types?

Reply:
Updated the description to note that there can be one or more media types on a single ad unit in d898dcb

Happy to update that description if you have additional comments!